ibtk/src/utilities/HierarchyIntegrator.cpp

```diff
@@ -263,6 +263,12 @@ HierarchyIntegrator::initializePatchHierarchy(Pointer<PatchHierarchy<NDIM> > hie
             done = !d_hierarchy->finerLevelExists(level_number);
             ++level_number;
         }
+
+        // Initialize composite hierarchy data that was not initailized by
+        // gridding algorithm's call to initializeLevelData().
+        initializeCompositeHierarchyData(d_integrator_time, initial_time);
+
```

Comment:
I think we have removed this call from here in MF branch. The reason is that it works fine is the data initialization is not dependent upon lag data. Some initiialization of composite hier may require lag data also

Reply:
This function should be called automatically by HierarchyIntegrator somewhere, so that implementations do not need to add that call explicitly.